src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -130,7 +132,8 @@ private static bool ShouldDisableInferredBodyParameters(string httpMethod)
     public ITypeSymbol Type { get; }
     public ITypeSymbol ElementType { get; }
 
-    public string Name { get; }
+    public string ParameterName { get; }
```

Comment:
`ParameterName` -> `SymbolName`

ParameterName isn't any clearer. Hopefully symbol name indicates to readers that the name came from the IParameterSymbol.